ui/app/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -114,15 +114,7 @@ function DateTimePicker(props: DateTimePickerProps) {
   const internalDate = useMemo(() => {
     const date = value ? new Date(value) : new Date();
     const localOffset = moment().format().substr(-6);
-    let dateWithoutOffset;
```

Comment:
Revert validation since converting is needed because the date is created in local timezone